modules/sdk/gradle-plugins-workspace/src/gradleTest/sourceEqualsDestCheck/build.gradle

```diff
@@ -0,0 +1,19 @@
+task runGradleTest
+
+gradle.liferayWorkspace {
+	bundleUrl = uri("bundlesDir/bundles.zip")
+}
+
+runGradleTest {
+	doLast {
+		try {
+			tasks.downloadBundle.execute()
+		}
+		catch (Exception e) {
+			assert downloadBundle.state.didWork == true
+			assert downloadBundle.state.executed == true
+			assert e.message.contains("Execution failed for task ':downloadBundle'")
```

Comment:
Lets make sure the exception is the same one that we threw containing the same message.  Its hard to see here if the exception is actually our exception.

Reply:
try `assert e.cause.message.contains("...")`